docs/features/feature-guides/logical-models/centralized-management.md

```diff
@@ -0,0 +1,68 @@
+# Centralized Management
+
+:::info
```

Comment:
You should add the Cloud only markdown tag here! This will add the little Cloud logo.